compositions/cluster-k8s/block-sync/latest/128-square-size/1-3-32-default.toml

```diff
@@ -0,0 +1,91 @@
+[metadata]
+  name = "blocksync-latest"
+  author = "derrandz"
+
+[global]
+  plan = "celestia"
+  case = "blocksync-latest"
+  total_instances = 36
+  builder = "docker:generic"
+  runner = "cluster:k8s"
+  disable_metrics = false
+
+# we define 1 validator that produces 8mb blocks
+# the msg-size does not result in exactly 8mb
```

Comment:
this is not 8mb blocks(msg size == 800kb). either put down eds 128 or say 0.8mb resulting in eds 128

Reply:
same for other compositions